### PR TITLE
#166722939 Change all freckle references to noko

### PR DESCRIPTION
**Pull request by bozicschucky**
#### What does this PR do?
Changes all occurrences of Noko to freckle.

#### Description of Task to be completed?
With Freckle rebranding to Noko, this task aims at renaming all occurrences of freckle to noko.

#### How should this be manually tested?
Yarn test

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#166722939](https://www.pivotaltracker.com/story/show/166722939)

#### Screenshots (If applicable)

### CODE REVIEW

**Comment by 09arnold** [comment]
Rename the image for the noko stats card to `noko.png`. The image isn't showing because of the mismatch

File: src/components/AutomationDetails/index.jsx [hunk 1]
```diff
@@ -115,64 +145,65 @@ class AutomationDetails extends PureComponent {
...
-
-  renderFreckleDetails = (modalContent) => {
-    const { freckleAutomations } = modalContent;
-    const freckleActivities = (freckleAutomations && freckleAutomations.freckleActivities) || [];
-    return (freckleActivities.map(content => (
+    ));
+  };
+
+  renderNokoDetails = (modalContent) => {
+    // console.log('-----> freckle modal content', modalContent);
```
Comment (katunold) [inline src/components/AutomationDetails/index.jsx]: Kindly do away with the commented-out code

File: src/components/developerCards/index.jsx [hunk 2]
```diff
@@ -72,20 +73,20 @@ class DeveloperCard extends Component {
 
   renderCards = () => {
     const { data, openModal, changeModalTypes } = this.props;
+    // console.log('card props --->>>>>', this.props);
```
Comment (katunold) [inline src/components/developerCards/index.jsx]: Kindly delete this commented out code

File: src/components/developerCards/index.jsx [hunk 3]
```diff
@@ -72,20 +73,20 @@ class DeveloperCard extends Component {
...
-    const automationMetric = (activities, status) => (
-      _.filter(activities, { status }).length
-    );
-    const metric = (activities, status) => (
-      automation(activities.length) ? automationMetric(activities, status) : 0
-    );
+    const automationMetric = (activities, status) => _.filter(activities, { status }).length;
+    const metric = (activities, status) => (automation(activities.length) ? automationMetric(activities, status) : 0);
     return data.map((card, index) => {
+      // console.log('card item and stuff --->>>>>', card);
```
Comment (katunold) [inline src/components/developerCards/index.jsx]: same here

File: src/components/developerCards/index.jsx [hunk 4]
```diff
@@ -72,20 +73,20 @@ class DeveloperCard extends Component {
...
+
       const name = card.fellowName;
       const newName = _.split(name, ',');
       const firstInitials = newName[0].charAt(0);
       const secondInitials = newName[1].charAt(1);
       return (
         <div className="card" key={card.id}>
+          {/* {console.log('card data -->>>>', card.nokoAutomations ? 'yooo' : 'data not avaiable')} */}
```
Comment (katunold) [inline src/components/developerCards/index.jsx]: And here

**Review by katunold — COMMENTED** [review]
I kindly suggest that you delete all the commented-out code, probably you forgot about it. But if you have reservations as to why you left the commented code then you can go ahead keep it in the codebase.

**Pull request closed without merging**